docs/api-call.md

```diff
@@ -1,55 +0,0 @@
-## Testing API for SUSE Manager
```

Comment:
i think you are pushing with a new pR the file? ok for me then

Reply:
No, this file is renamed "cucumber-steps.md"

All the old contents are preserved, of course, and a lot added.

Reply:
ok